services/document-manager/backend/app/services/object_store_storage_service.py

```diff
@@ -1,9 +1,32 @@
-import boto3
 import io
+import os
+import sys
+import threading
+import boto3
+import hashlib
+
+from botocore.exceptions import ClientError
 from flask import send_file, Response, current_app
+
 from app.config import Config
 
 
+class ProgressPercentage(object):
```

Comment:
Just a note, I added this in for fun/out of curiosity to see progress for large file uploads, but it's turned off to prevent clogging up the logfile.

Reply:
ok :)